Source/OnlineSubsystemEpic/Private/OnlineIdentityInterfaceEpic.cpp

```diff
@@ -232,12 +227,14 @@ void FOnlineIdentityInterfaceEpic::LoginCompleteCallbackFunc(const EOS_Auth_Logi
 				break;
 			}
 		}
+		auto idString = id.ToString();
+		if (!thisPtr->userAccounts.Contains(idString)) {
```

Comment:
repeated calls to login would cause duplicate keys with the same shared ref, which was causing some very strange access violation bugs (unsure of the technical reason, maybe a duplicate key issue?) adding this check makes the login process idompotent